IHP/IDE/SchemaDesigner/View/Columns/Edit.hs

```diff
@@ -113,38 +113,47 @@ instance View EditColumnView where
 typeSelector :: Maybe PostgresType -> [Text] -> Html
 typeSelector postgresType enumNames = [hsx|
         <select id="typeSelector" name="columnType" class="form-control select2-simple">
-            {option selected "TEXT" "Text"}
-            {option selected "INT" "Int"}
-            {option selected "UUID" "UUID"}
-            {option selected "BOOLEAN" "Bool"}
-            {option selected "TIMESTAMP WITH TIME ZONE" "Timestamp (UTCTime)"}
-            {option selected "TIMESTAMP WITHOUT TIME ZONE" "Timestamp (LocalTime)"}
-            {option selected "REAL" "Float"}
-            {option selected "DOUBLE PRECISION" "Double"}
-            {option selected "POINT" "Point"}
-            {option selected "DATE" "Date"}
-            {option selected "BYTEA" "Binary"}
-            {option selected "Time" "Time"}
-            {option selected "SERIAL" "Serial"}
-            {option selected "BIGSERIAL" "Bigserial"}
-            {option selected "SMALLINT" "Int (16bit)"}
-            {option selected "BIGINT" "Int (64bit)"}
-            {option selected "JSONB" "JSON"}
-            {forEach enumNames renderEnumType}
+            <optgroup label="Common Types">
+                {option isSelected "TEXT" "Text"}
+                {option isSelected "INT" "Int"}
+                {option isSelected "UUID" "UUID"}
+                {option isSelected "BOOLEAN" "Bool"}
+                {option isSelected "DATE" "Date / Day"}
+                {option isSelected "TIMESTAMP WITHOUT TIME ZONE" "Timestamp (LocalTime)"}
+                {option isSelected "SERIAL" "Serial"}
+            </optgroup>
+            {customenums enumNames}
+            <optgroup label="Other Types">
+                {option isSelected "TIMESTAMP WITH TIME ZONE" "Timestamp (UTCTime)"}
```

Comment:
`TIMESTAMP WITH TIME ZONE` is the default type used for `created_at` fields in the schema designer (or generally any field whose name ends with `_at`). I think therefore we should have it in Common Types.

[I'm a big fan of storing all timestamps in the db in UTC](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29_to_store_UTC_times). Any reasons we cannot move the `TIMESTAMP WITHOUT TIME ZONE` into `Other Types` to `Timestamp UTCTime` the default suggested timestamp type?

Reply:
No problem, pushed change now. As long as my types are there, it isn't important where they are in the list. I'm more of a localtime (or time with actual timezone) kind of guy. UTC has given me unnecessary complexity in the past. But UTC is a good choice for created_at etc.